test/tools/yulInterpreter/EVMInstructionInterpreter.cpp

```diff
@@ -468,6 +468,8 @@ u256 EVMInstructionInterpreter::evalBuiltin(
 			);
 		return 0;
 	}
+	else if (fun == "linkersymbol")
+		assertThrow(false, YulException, "The bytecode is incomplete and contains unresolved link references.");
```

Comment:
`yulAssert` also emits `YulException`.

Reply:
I know. But I wanted it to look like a validation (or a runtime error?), not an assertion. It can happen at runtime.

Maybe I should introduce a new exception type here?